erpnext/projects/doctype/project/project.js

```diff
@@ -80,6 +80,15 @@ frappe.ui.form.on("Project", {
 		frm.events.set_buttons(frm);
 	},
 
+	onload_post_render: function(frm) {
+		$(".menu-item-label[data-label='Duplicate']")
+			.parent('a')
+			.unbind("click")
+			.bind("click", function() {
+				frm.trigger('create_duplicate_project');
```

Comment:
Function definition is `create_duplicate`?????

Reply:
Added changes